daemon/src/projection.rs

```diff
@@ -810,12 +826,21 @@ impl From<Order> for CfdOrder {
                 .settlement_interval
                 .whole_seconds()
                 .try_into()
-                .expect("settlement_time_interval_hours is always positive number"),
+                .context("unable to convert settlement interval")?,
             opening_fee: Some(order.opening_fee.to_inner()),
-            funding_rate_annualized_percent: AnnualisedFundingRate::from(order.funding_rate)
+            funding_rate_annualized_percent: AnnualisedFundingPercent::from(order.funding_rate)
                 .to_string(),
-            funding_rate_hourly_percent: HourlyFundingRate::from(order.funding_rate).to_string(),
-        }
+            funding_rate_hourly_percent: HourlyFundingPercent::from(order.funding_rate).to_string(),
+            initial_funding_fee_per_parcel: calculate_funding_fee(
+                order.price,
+                parcel_size,
+                order.leverage,
+                order.funding_rate,
+                SETTLEMENT_INTERVAL.whole_hours(),
+            )
+            .context("unable to calcualte initial funding fee")?
```

Comment:
Do we have test cases that show when this can fail?

Reply:
https://github.com/itchysats/itchysats/pull/1255 should cover this, but will need to be adapted to my changes.
I'd say this is not critical for the release and can be done later.